cmd/agent/commands/grpc.go

```diff
@@ -32,6 +33,8 @@ func BuildGrpcCmd(env runtime.Environment, config *agent.Config) *cobra.Command
 			listenAddress := fmt.Sprintf(":%d", port)
 			upstreamAddress := fmt.Sprintf("%s:%d", upstreamHost, target)
 
+			var err error
```

Comment:
Why do we pre-declare `err` here? It seems to be shadowed on line 43.

Reply:
Maybe is a left-over of some code reorganization.